knowledge-base/actions/pragatheeswarans/yarn-audit-action/action-security.yml

```diff
@@ -0,0 +1,11 @@
+name: "Yarn Audit"
+github-token:
+  action-input:
+    input: GITHUB_TOKEN
```

Comment:
Please update input to `input: token` view [here](https://github.com/pragatheeswarans/yarn-audit-action/blob/f3ab4c055f4b8f1db1b1ae0e32bab690c9050603/action.yml#L7)